pkg/downloader/manager_test.go

```diff
@@ -360,3 +360,32 @@ func TestBuild_WithRepositoryAlias(t *testing.T) {
 		Repository: "@test",
 	})
 }
+
+func TestErrRepoNotFound_Error(t *testing.T) {
+	type fields struct {
+		Repos []string
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   string
+	}{
+		{
+			name: "OK",
+			fields: fields{
+				Repos: []string{"https://charts1.example.com", "https://charts2.example.com"},
+			},
+			want: "no repository definition for https://charts1.example.com, https://charts2.example.com",
```

Comment:
Can you please also add a test case for the issue we discussed earlier, where only a few repositories in the list are missing.